### PR TITLE
slash command mentions

### PR DESCRIPTION
**Pull request by SuperchupuDev**
this is

### CODE REVIEW

File: i18n/pl.json [hunk 1]
```diff
@@ -144,7 +143,7 @@
     "PROFILE_INVALID_WEBSITE": "Podałeś nieprawidłowy link! Przykłady prawidłowych linków: https://example.com, http://example.com, example.com\nMaksymalna długość linku: 25 znaków!",
     "PROFILE_LOCATION": "Lokalizacja",
     "PROFILE_NOT_SET": "Nie ustawiono!",
-    "PROFILE_NOT_SET_DESCRIPTION": "Nie ustawiono! Ustaw swój profil za pomocą /profile set",
+    "PROFILE_NOT_SET_DESCRIPTION": "Nie ustawiono! Ustaw swój profil za pomocą /profil ustaw",
```
Comment (Nyameliaaaa) [inline i18n/pl.json:146]: this should use slash command mentions

**Review by Nyameliaaaa — CHANGES_REQUESTED** [review]

File: i18n/pl.json (hunk 1 above)
Comment (SuperchupuDev) [reply i18n/pl.json:146]: that string isnt rendered inside discord, its the one that shows up in the /profile view image

**Review by Nyameliaaaa — APPROVED** [review]
sure

**Comment by SuperchupuDev** [comment]
thanks

**Pull request merged**